package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.2.0",
   "private": true,
   "engines": {
-    "node": "12.13.1"
+    "node": "^12.13.1"
```

Comment:
This relaxes the NodeJS dependency for deployments via Heroku. Is that what you intended?

Heroku has the right version anyway and will also install the same version 12.13.1 even with your change:

```
munen@lambda:~/src/200ok/organice% git push heroku master
Enumerating objects: 8868, done.
Counting objects: 100% (8868/8868), done.
Delta compression using up to 12 threads
Compressing objects: 100% (4504/4504), done.
Writing objects: 100% (8868/8868), 3.88 MiB | 553.00 KiB/s, done.
Total 8868 (delta 5962), reused 6107 (delta 4135)
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: -----> Node.js app detected
remote:        
remote: -----> Creating runtime environment
remote:        
remote:        NPM_CONFIG_LOGLEVEL=error
remote:        NODE_ENV=production
remote:        NODE_MODULES_CACHE=true
remote:        NODE_VERBOSE=false
remote:        
remote: -----> Installing binaries
remote:        engines.node (package.json):  12.13.1
```

Reply:
Ok, turns out that `yarn` also reads this if it is set. Hence, I'll retract my comment.